packages/design-system/src/Switch/Switch.styles.tsx

```diff
@@ -2,7 +2,19 @@ import styled, { css } from "styled-components";
 import { Text } from "../Text";
 
 const Variables = css`
-  --switch-color-bg: var(--ads-v2-color-bg-emphasis);
+  --ads-v2-colors-control-switch-track-default-bg: var(
+    --ads-v2-color-bg-emphasis
+  );
+  --ads-v2-colors-control-switch-track-hover-bg: var(
+    --ads-v2-color-bg-emphasis-plus
+  );
+  --ads-v2-colors-control-switch-track-checked-bg: var(
+    --ads-v2-color-bg-brand-secondary
+  );
+  --ads-v2-colors-control-switch-track-checked-hover-bg: var(
+    --ads-v2-color-bg-brand-secondary-emphasis
+  );
+
   --thumb-color-bg: var(--ads-v2-color-bg);
```

Comment:
Let's keep `--switch-color-bg` and have the category tokens change the value of the component token only